### ISSUE TITLE
44_sess.t sometimes fails with older Perls on Windows

### ISSUE THREAD

**Issue by h-vn** [radiator-software/p5-net-ssleay#262]
AppVeyor and CPAN testers reports show that `44_sess.t` sometimes hangs or fails on Windows. Failures are typically seen on Strawberry Perl versions 5.14, 5.16 and 5.18. The failures seem to be related to networking problems between two forked instances. A closer look showed that `close()` was not called but a new socket was assigned to the same scalar variable in al oop that test uses. While Perl documentation tells that `close()` is automatically called when the socket goes out of scope, calling `close()` explicitly before starting a new loop round appears to fix the problem. What was also noticed is that even a simple debug prints changed something (timing?) allowing the test to succeed much more often.

Calling `close(`) explicitly is better in any case because it allows for error checking.

This problem was seen also with a socket that was opened and used without SSL/TLS. In other words, the problem is not related to SSL/TLS.
